src/main/java/seedu/duke/CommandList.java

```diff
@@ -35,7 +41,13 @@ public static void executeShoot(String[] readArgumentTokens) {
     public static void executePenalty() {
         Penalty.executePenalty();
     }
-    
+    public static void executeUpgrade(String[] level){
+        String upgradeLevel = level[0];
+        int upgradeLevelIndex = Integer.parseInt(upgradeLevel);
+
+        PlayerList.L1.get(0).upgradePower(upgradeLevelIndex);
+        PlayerList.L1.get(0).printSelfInfo();
+    }
```

Comment:
I think the player stats should be upgraded automatically. It shouldn't be a command.

Reply:
There are two attributes for player, one is skill, and the other is power.
Skill is automatically upgraded with the player when it plays enough number of game
For power, user can choose to upgrade manually, even at the beginner level.